itsy.css

```
@@ -0,0 +1,87 @@
+body {
+  margin: 0px;
+}
+header {
+  display: flex;
+  background: url(img/lead.jpg);
+  height: 100vh;
+  flex-direction: column;
+  justify-content: space-between;
+
+}
+
+.header-top {
+  display: flex;
+  justify-content: space-between;
+  padding: 10px 30px;
```

looks a little bit better

```suggestion
  padding: 20px;
```

---

not , i like my varriant